### ISSUE TITLE
Aria Support?

### ISSUE THREAD

**Issue by nadnoslen** [shipshapecode/ember-flatpickr#306]
Just did a quick search of the issues and don't see any mention of aria support.  Wonder if I can help fold a few of those aria attributes into these inputs?  e.g. `aria-describedby`.

**Comment by rwwagner90** [comment]
@nadnoslen we would love the help! I am not sure the state of aria and a11y support in general in flatpickr and not sure if this would be better here or there, but totally agree it should be supported!

**Comment by nadnoslen** [comment]
I'll take a look at the component implementation to just see if we can wedge it into the input without doing any damage.

**Comment by rwwagner90** [comment]
@nadnoslen do you think it belongs here, rather than in flatpickr itself?

**Comment by nadnoslen** [comment]
Hey @rwwagner90, your wonderful add-on has abstracted so much of Flatpickr away from me that I'm not yet qualified to answer that question!!

In my opinion, it aria attribute assignments would be responsible by the code that is actually defining the `<input>`.

I've forked your addon and will poke around shortly.  😉

**Comment by nadnoslen** [comment]
Hey @rwwagner90,

Looks like adding a few more `attributeBindings` is what is in order.  I'm working on this in my fork.  I'll summarize when I'm done.  Cheers.

**Comment by rwwagner90** [comment]
Sounds good 👍
